std/datetime/date.d

```diff
@@ -146,7 +146,7 @@ public:
     /++
         Params:
             year   = The year portion of the date.
-            month  = The month portion of the date.
+            month  = The month portion of the date (January is 1).
             day    = The day portion of the date.
             hour   = The hour portion of the time;
             minute = The minute portion of the time;
```

Comment:
I think that's a little overkill? The only other one that *maybe* might be a little ambiguous is the day of month (as it's the only 1-based one). Doing it consistently would also require updating the documentation anywhere these occur as integers (which in the case of everything except months and days of the week, is pretty much everywhere).